src/services/posts/fetch-posts.js

```diff
@@ -0,0 +1,36 @@
+const db = require("../../db")
+
+const fetchPosts = async(isFlagged) => {
+
+	
+	let selectClause = `
+	SELECT posts.id, posts."userId" , posts.title, posts.image,
+	posts.content,posts.published, posts."createdAt", posts.type,
+	posts."likesCount",
+	users."firstName", users."lastName", users.email,
+	users."profilePictureUrl" 
+	`
+
+	 const fromClause = `FROM posts`
+	  
+	 const joinClause = `INNER JOIN users on posts."userId" = users.id`
+	 let whereClause = ''
+	 let orderByClause = `ORDER BY posts."createdAt" DESC`
+	   
+	 if(isFlagged !== undefined){
+		 selectClause+= `, posts."flagsCount"`
+		 
+		whereClause += `
+			WHERE posts."flagsCount" ${isFlagged ? '>': '='} 0`
+		orderByClause+=`ORDER BY "flagsCount" DESC`
```

Comment:
after the most recent change, this will no longer be a concatenation, just an assignment to replace the default order you have added

```suggestion
		orderByClause =`ORDER BY "flagsCount" DESC`
```

Reply:
I have made all suggested changes, kindly review, thanks